RELEASE/scripts/postsool.ash

```diff
@@ -757,6 +760,32 @@ void handlePostAdventure()
 				buffMaintain($effect[Ceaseless Snarling], 0, 1, 10);
 			}
 		}
+
+		// If we are in some state where we do not want +ML (Level 13 or Smut Orc) make sure ML is removed
+		if(removeML)
+		{
+			if(0 < have_effect($effect[Driving Recklessly]))
+			{
+				uneffect(Driving Recklessly);
+			}
+			if(0 < have_effect($effect[Ur-Kel\'s Aria of Annoyance]))
+			{
+				uneffect(Ur-Kel\'s Aria of Annoyance);
+			}
+			if(0 < have_effect($effect[Drescher\'s Annoying Noise]))
+			{
+				uneffect(Drescher\'s Annoying Noise);
```

Comment:
I don't know if I agree, especially if the player does not have Asdon Martin.

Smut Orcs have a base 69 HP.
+ML adds +1 HP per level, subtracts 1 per negative level.
Blech House manifests when 15 Orc worth of damage is done, capped at 5x per kill.

So without buffs you need to do:
2x = 138
3x = 207
4x = 276
5x = 345

Letting both of these ride, no Asdon (HP=89):
2x = 178
3x = 267
4x = 356
5x = 445

Dropping these AND having the Asdon Active (HP=44):
2x = 88
3x = 132
4x = 176
5x = 220

Without knowing exactly what each player has for equips and permed from the list I can't quantify this into turn savings, but by eliminating those (if we can) it makes it more low-IOTM and low-skill friendly IMO, and still benefits high-IOTM and high-PERM players, especially running in a non-MP friendly path.

Would maybe "sl_aggressiveML = false" on the relay be better and let the user decide?

Reply:
Also, I should note that +ML increases Monster Defense.

Reply:
Actually maybe you're right, but it's still not as important as being able to uneffect on the trail when needed. It's okay for people swimming in SGEEAs or without access to Transcendent Olfaction though (either because they don't have it or they're on a path without normal skill access), so how about with this conditional:

`if(!sl_have_skill($skill[Transcendent Olfaction]) || item_amount($item[soft green echo eyedrop antidote]) > 5)`

around the block with uneffecting the things that take sgeeas?